config/runtime.exs

```diff
@@ -163,25 +179,13 @@ delete_brainstormings_after_days =
     30
   end
 
-feature_file_upload =
-  Enum.member?(
-    ["", "true"],
-    String.trim(System.get_env("MW_FEATURE_IDEA_FILE_UPLOAD") || "")
-  )
+feature_file_upload = EnvHelper.enabled?("MW_FEATURE_IDEA_FILE_UPLOAD", "true")
 
-feature_privacy_imprint_enabled =
-  Enum.member?(
-    ["true"],
-    String.trim(System.get_env("MW_FEATURE_LEGAL_PRIVACY_LINKS") || "")
-  )
+feature_privacy_imprint_enabled = EnvHelper.enabled?("MW_FEATURE_LEGAL_PRIVACY_LINKS", "false")
```

Comment:
this minimally changes behavior, I guess the reason it was different from the others (Checking only against "true" and not "") was so that it was disabled by default/if no env was set.

By default it will still be disabled. However, if someone has set an empty environment variable this will not cause it to be activated. This is in line with all the others, so I think that's a win for consistency :)